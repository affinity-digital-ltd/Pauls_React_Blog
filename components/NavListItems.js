import { Component } from 'react'

class NavListItems extends Component {
  render () {
    return <div>
      <ul className='c-footer__nav'>
        <li><a href='#'>Home</a></li>
        <li><a href='#'>About</a></li>
        <li><a href='#'>Portfolio</a></li>
        <li><a href='#'>Contact</a></li>
      </ul>
    </div>
  }
}

export default NavListItems
