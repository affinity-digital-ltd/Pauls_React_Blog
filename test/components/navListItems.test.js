import NavListItems from '../../components/NavListItems'

describe('<NavListItems />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <NavListItems />
    )

    expect(component).toMatchSnapshot()
  })
})
