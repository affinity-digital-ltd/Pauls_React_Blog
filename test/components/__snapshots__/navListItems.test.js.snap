// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavListItems /> it renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavListItems />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <li>
          <Link
            href="/index"
          >
            <a
              onClick={undefined}
            >
              Home
            </a>
          </Link>
        </li>,
        <li>
          <Link
            href="/about"
          >
            <a
              onClick={undefined}
            >
              About
            </a>
          </Link>
        </li>,
        <li>
          <Link
            href="/portfolio"
          >
            <a
              onClick={undefined}
            >
              Portfolio
            </a>
          </Link>
        </li>,
        <li>
          <Link
            href="/contact"
          >
            <a
              onClick={undefined}
            >
              Contact
            </a>
          </Link>
        </li>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            href="/index"
          >
            <a
              onClick={undefined}
            >
              Home
            </a>
          </Link>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <a
              onClick={undefined}
            >
              Home
            </a>,
            "href": "/index",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Home",
              "onClick": undefined,
            },
            "ref": null,
            "rendered": "Home",
            "type": "a",
          },
          "type": [Function],
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            href="/about"
          >
            <a
              onClick={undefined}
            >
              About
            </a>
          </Link>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <a
              onClick={undefined}
            >
              About
            </a>,
            "href": "/about",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "About",
              "onClick": undefined,
            },
            "ref": null,
            "rendered": "About",
            "type": "a",
          },
          "type": [Function],
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            href="/portfolio"
          >
            <a
              onClick={undefined}
            >
              Portfolio
            </a>
          </Link>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <a
              onClick={undefined}
            >
              Portfolio
            </a>,
            "href": "/portfolio",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Portfolio",
              "onClick": undefined,
            },
            "ref": null,
            "rendered": "Portfolio",
            "type": "a",
          },
          "type": [Function],
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            href="/contact"
          >
            <a
              onClick={undefined}
            >
              Contact
            </a>
          </Link>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <a
              onClick={undefined}
            >
              Contact
            </a>,
            "href": "/contact",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Contact",
              "onClick": undefined,
            },
            "ref": null,
            "rendered": "Contact",
            "type": "a",
          },
          "type": [Function],
        },
        "type": "li",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li>
            <Link
              href="/index"
            >
              <a
                onClick={undefined}
              >
                Home
              </a>
            </Link>
          </li>,
          <li>
            <Link
              href="/about"
            >
              <a
                onClick={undefined}
              >
                About
              </a>
            </Link>
          </li>,
          <li>
            <Link
              href="/portfolio"
            >
              <a
                onClick={undefined}
              >
                Portfolio
              </a>
            </Link>
          </li>,
          <li>
            <Link
              href="/contact"
            >
              <a
                onClick={undefined}
              >
                Contact
              </a>
            </Link>
          </li>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              href="/index"
            >
              <a
                onClick={undefined}
              >
                Home
              </a>
            </Link>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <a
                onClick={undefined}
              >
                Home
              </a>,
              "href": "/index",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Home",
                "onClick": undefined,
              },
              "ref": null,
              "rendered": "Home",
              "type": "a",
            },
            "type": [Function],
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              href="/about"
            >
              <a
                onClick={undefined}
              >
                About
              </a>
            </Link>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <a
                onClick={undefined}
              >
                About
              </a>,
              "href": "/about",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "About",
                "onClick": undefined,
              },
              "ref": null,
              "rendered": "About",
              "type": "a",
            },
            "type": [Function],
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              href="/portfolio"
            >
              <a
                onClick={undefined}
              >
                Portfolio
              </a>
            </Link>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <a
                onClick={undefined}
              >
                Portfolio
              </a>,
              "href": "/portfolio",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Portfolio",
                "onClick": undefined,
              },
              "ref": null,
              "rendered": "Portfolio",
              "type": "a",
            },
            "type": [Function],
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              href="/contact"
            >
              <a
                onClick={undefined}
              >
                Contact
              </a>
            </Link>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <a
                onClick={undefined}
              >
                Contact
              </a>,
              "href": "/contact",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Contact",
                "onClick": undefined,
              },
              "ref": null,
              "rendered": "Contact",
              "type": "a",
            },
            "type": [Function],
          },
          "type": "li",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
